### PR TITLE
NIFI-11049 Fixed flaky PutDropboxTest, attribute description corrected

### PR DESCRIPTION
**Pull request by krisztina-zsihovszki**
# Summary

[NIFI-11049](https://issues.apache.org/jira/browse/NIFI-11049)

# Tracking

Please complete the following tracking steps prior to pull request creation.

### Issue Tracking

- [x] [Apache NiFi Jira](https://issues.apache.org/jira/browse/NIFI) issue created

### Pull Request Tracking

- [x] Pull Request title starts with Apache NiFi Jira issue number, such as `NIFI-11049`
- [x] Pull Request commit message starts with Apache NiFi Jira issue number, as such `NIFI-11049`

### Pull Request Formatting

- [x] Pull Request based on current revision of the `main` branch
- [x] Pull Request refers to a feature branch with one commit containing changes

# Verification

Please indicate the verification steps performed prior to pull request creation.

### Build

- [ ] Build completed using `mvn clean install -P contrib-check`
  - [x] JDK 8
  - [x] JDK 11
  - [x] JDK 17

### Licensing

- [ ] New dependencies are compatible with the [Apache License 2.0](https://apache.org/licenses/LICENSE-2.0) according to the [License Policy](https://www.apache.org/legal/resolved.html)
- [ ] New dependencies are documented in applicable `LICENSE` and `NOTICE` files

### Documentation

- [x] Documentation formatting appears as expected in rendered files

### CODE REVIEW

File: nifi-nar-bundles/nifi-dropbox-bundle/nifi-dropbox-processors/src/test/java/org/apache/nifi/processors/dropbox/PutDropboxTest.java [hunk 1]
```diff
@@ -255,18 +258,18 @@ void testFileUploadLargeFile() throws Exception {
                 .thenReturn(mockUploadSessionAppendV2Uploader);
 
         //finish session: 30 - 8 - 2 * 8 = 6 bytes uploaded
-        CommitInfo commitInfo = CommitInfo.newBuilder(getPath(TEST_FOLDER , FILENAME_1))
+        CommitInfo commitInfo = CommitInfo.newBuilder(getPath(TEST_FOLDER, FILENAME_1))
```
Comment (nandorsoma) [inline nifi-nar-bundles/nifi-dropbox-bundle/nifi-dropbox-processors/src/test/java/org/apache/nifi/processors/dropbox/PutDropboxTest.java:261]: I don't feel strongly, but I wouldn't recreate this object solely to have a prototype that can be used as a matcher. Plain any() matcher would be enough for me, but if we wanted to match its properties, I would create a Factory, especially for CommitInfo, which we can mock adequately.

**Review by nandorsoma — APPROVED** [review]
Hey @krisztina-zsihovszki! Thank you for your fix, LGTM! I've found one minor thing, feel free to ignore it if you don't agree.

**Review by nandorsoma — APPROVED** [review]
Thanks for the additional changes!

**Comment by turcsanyip** [comment]
@nandorsoma Thanks for your review!

Merging to main...

**Pull request closed without merging**